packages/react-native-editor/package.json

```diff
@@ -34,7 +34,6 @@
 		"@react-native-community/blur": "4.2.0",
 		"@react-native-community/slider": "https://raw.githubusercontent.com/wordpress-mobile/react-native-slider/v3.0.2-wp-4/react-native-community-slider-3.0.2-wp-4.tgz",
 		"@react-native-masked-view/masked-view": "0.3.0",
-		"@react-native/gradle-plugin": "0.72.11",
```

Comment:
This is removed since React Native's has it as a dependency which supports the current AGP version.